themes/default/content/docs/reference/service-api/_index.md

```diff
@@ -0,0 +1,1465 @@
+---
+title: "Pulumi Service API"
+meta_desc: An overview of the Pulumi Service API for querying Organization, Stack, State, etc. information.
+---
+
+The Pulumi Service API is used by the Pulumi CLI and the Pulumi Console to query and interact with state information, history, stack tags, etc. This API is available for end users to integrate into their own automation use cases.
+
+{{% notes "info" %}}
+While the Pulumi Service API is used by the Pulumi CLI and Pulumi Console, Pulumi makes no guarantees about changes that might affect backwards compatibility or cause breaking changes.
+{{% /notes %}}
+
+## Authentication
+
+All requests must be authenticated using a token via the `Authorization` HTTP header. If the token is missing or invalid, the Pulumi Service API will return a 4xx status code.
+
+The `Authorization` header must be in the form below with the literal string `token`, then a space, then your access token value.
```

Comment:
This is fine as-is, but we may be able to rephrase this to sound less like we just made this up and point to the docs for the Authorization header and it's ~standard.
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Authorization

So the "authorization type" is `token`. But again, this is fine as-is.

Reply:
Going to leave as-is per our conversation.